public/app/plugins/panel/canvas/SetBackground.tsx

```diff
@@ -0,0 +1,67 @@
+import { css } from '@emotion/css';
+import React, { useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Portal, useTheme2 } from '@grafana/ui';
+import { Scene } from 'app/features/canvas/runtime/scene';
+
+import { MediaType, ResourceDimensionMode, ResourceFolderName } from '../../../features/dimensions';
+import { ResourcePickerPopover } from '../../../features/dimensions/editors/ResourcePickerPopover';
+
+import { AnchorPoint } from './types';
+
+type Props = {
+  onClose: () => void;
+  scene: Scene;
+  anchorPoint: AnchorPoint;
+};
+
+export function SetBackground({ onClose, scene, anchorPoint }: Props) {
+  const defaultValue = scene.root.options.background?.image?.fixed ?? '';
+
+  const [bgImage, setBgImage] = useState(defaultValue);
+  const theme = useTheme2();
+  const styles = getStyles(theme, anchorPoint);
+
+  const onChange = (value: string | undefined) => {
+    if (value) {
+      setBgImage(value);
+      if (scene.root) {
+        scene.root.options.background = {
+          ...scene.root.options.background,
+          image: { mode: ResourceDimensionMode.Fixed, fixed: value },
```

Comment:
Note: for context menu set background option we are only supporting setting a fixed image vs an image based on a field value